src/main/resources/archetype-resources/silo/src/main/resources/application.yml

```diff
@@ -23,7 +23,7 @@ spring:
         name: ${rootArtifactId}
 
     flyway:
-        locations: classpath:db/migration, classpath:db/common
+        locations: classpath:db
```

Comment:
Would still prefer having `db/common`, otherwise it will be quite difficult to ever have something which is only relevant for testing / migration or something in that direction.